stdlib/3/tempfile.pyi

```diff
@@ -19,9 +19,9 @@ template: str
 
 if sys.version_info >= (3, 6):
     from builtins import _PathLike
-    _PathType = Union[_PathLike[AnyStr], AnyStr]
+    _PathType: Union[_PathLike[AnyStr], AnyStr]
```

Comment:
I don't think this is the right fix. This changes things completely, from "`_PathType` is a type alias for such and such a type" to "`_PathType` is a variable whose type is such and such a type".

Unfortunately, since `AnyStr` is a type variable, using it in a type alias is also not allowed (the problem you are trying to solve here). I'm not sure if it can be solved -- you may have to repeat all affected definitions a second time, once for 3.6 and once for older versions.

Or you could satisfy yourself with a type declaration that's a bit of a lie for Python 3.5 -- that version's End of Life is projected soon, when 3.8.0 comes out (late October).

Reply:
You can use `_PathType = Union[AnyStr]` hack.

Reply:
Also use a newly defined type variable here, otherwise pytype test will fail (anyway using `AnyStr` in a generic alias is misleading, since type variables are totally dummy within generic aliases).

Reply:
I had thought of doing like that but I was hoping there was a less ugly way of doing it.